packages/core/rpc/configSchema.ts

```diff
@@ -1,20 +1,27 @@
 import { types } from 'mobx-state-tree'
 import { ConfigurationSchema } from '../configuration'
 
+const BaseRpcDriverConfigSchema = ConfigurationSchema(
+  'MainThreadRpcDriver',
```

Comment:
This might create 2 configs named MainThreadRpcDriver?

Reply:
Oops, good catch. Copy-paste error, fixed now.